README.md

```diff
@@ -1,2 +1,3 @@
-# Glykon
-glykon offers an immersive experiance that is fully open source. No premium ever.
+<h1  align="center">Glykon</h1>
+<div align="center">Glykon offers an immersive experience that is <b>fully open source. No premium ever.</b></div>
```

Comment:
Whats the point of the bold here?

Reply:
I don't see adding bold there as a too useful thing, it looks ok tho.